src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -3638,3 +3649,81 @@ class ValuesClauseSegment(BaseSegment):
         # TO DO - CHECK OFFSET
         # TO DO - FETCH
     )
+
+
+@postgres_dialect.segment()
+class DeleteUsingClauseSegment(BaseSegment):
+    """USING clause."""
+
+    type = "using_clause"
+    match_grammar = StartsWith(
+        "USING",
+        terminator="WHERE",
+        enforce_whitespace_preceding_terminator=True,
+    )
+
+    parse_grammar = Sequence(
+        "USING",
+        Indent,
+        # NB: We don't use BracketedColumnReferenceListGrammar
```

Comment:
I'm having trouble relating this comment to the code. The code uses `TableExpressionSegment`, but the comment says we're using `SingleIdentifierGrammar`. 🤯

Reply:
Remove this whole comment. Copied over from code I stole this from.